ui/src/message_popup/playtest/danson_experience_page.jsx

```diff
@@ -0,0 +1,344 @@
+/* @flow weak */
```

Comment:
It might be good to add a single test for this new experience page component.  That would catch things like if we change the API to `PopupQuestion` (like we took advantage of today).  There's a model [here](https://github.com/mit-teaching-systems-lab/threeflows/blob/master/ui/src/message_popup/demo_page_test.jsx) if you have time and want to do that tomorrow, but that doesn't need to hold up this pull request.